### PR TITLE
Refactor: StatsD code removal

### PR DESCRIPTION
**Pull request by sumslim**
OpenTelemetry is being used now for sending metrics to Graphite, removing redundant StatsD code

### CODE REVIEW

**Comment by bsushmith** [comment]
@sumslim can you fix the test case failures.

**Comment by sumslim** [comment]
> @sumslim can you fix the test case failures.

Yes @bsushmith working on that

File: cmd/run.go [hunk 1]
```diff
@@ -71,15 +71,6 @@ func RunCmd() *cobra.Command {
 			defer stop()
 
 			var mts []agent.Monitor
```
Comment (haveiss) [inline cmd/run.go]: we no longer requiring a list now since only otel supporterd?

Comment (sumslim) [reply cmd/run.go]: @haveiss this I wanted to discuss with you, just kept it list for extensibility and future support
if we want to add couple of another metrics in the future, not much code change would be required
any suggestion on this?

Comment (haveiss) [reply cmd/run.go]: I don't think we will need another one in foreseeable future, OpenTelemetry is already became a standard, it is quite extensible to anything we need

Comment (sumslim) [reply cmd/run.go]: Okay sure, removing the list

**Review by Mayurjag — APPROVED** [review]
lgtm

**Review by haveiss — APPROVED** [review]

**Review by bsushmith — APPROVED** [review]

**Pull request merged**